packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -24,21 +24,22 @@ import {Subject} from 'rxjs'
 import {Box, useToast} from '@sanity/ui'
 import scrollIntoView from 'scroll-into-view-if-needed'
 import {debounce} from 'lodash'
-import {FormPatch, SANITY_PATCH_TYPE} from '../../patch'
+import {SANITY_PATCH_TYPE} from '../../patch'
 import {ArrayOfObjectsItemMember, ObjectFormNode} from '../../store'
 import type {ArrayOfObjectsInputProps, PortableTextMarker, RenderCustomMarkers} from '../../types'
 import {EMPTY_ARRAY} from '../../../util'
 import {pathToString} from '../../../field'
 import {isMemberArrayOfObjects} from '../../members/object/fields/asserters'
 import {FormInput} from '../../components'
 import {FIXME} from '../../../FIXME'
+import {useFormCallbacks} from '../../studio'
 import {Compositor, PortableTextEditorElement} from './Compositor'
 import {InvalidValue as RespondToInvalidContent} from './InvalidValue'
 import {usePatches} from './usePatches'
 import {RenderBlockActionsCallback} from './types'
 import {PortableTextMarkersProvider} from './contexts/PortableTextMarkers'
 import {PortableTextMemberItemsProvider} from './contexts/PortableTextMembers'
-import {_isArrayOfObjectsFieldMember, _isBlockType} from './_helpers'
```

Comment:
potential nitpick but a thought: I understand that we are now making them explicitly `internal` on its definition, but shouldn't we keep the naming convention to more easily identify it as internal still?

Reply:
> potential nitpick but a thought: I understand that we are now making them explicitly `internal` on its definition, but shouldn't we keep the naming convention to more easily identify it as internal still?

Good question. Do we even use this naming convention still? I think it looks very ugly. And we got TS to take care of it like you say.

Reply:
~~I also don't think it looks pretty don't get me wrong 😬~~ 
I'm only asking because I myself had this question somewhat recently but ended up not needing to do it.
@sanity-io/studio-dx we might move this discussion out of github but quick thoughts on this (if it's defined as `@internal` in tsdocs then perhaps we don't need to keep the naming convention for internal things)?

Reply:
👋 @skogsmaskin just had a chat with the team and the consensus is as long as it’s tagged with `@internal` then the naming convention is dropped. 

marking this as resolved ✨